homestyle.css

```diff
@@ -42,35 +42,23 @@ body {
 }
 
 .links a {
-  background-color: black;
-  border: #565656 solid 5px
-  color: white;
+  border: #000000 solid 5px;
   padding: 15px 50px;
   text-align: center;
   text-decoration: none;
   font-size: 20px;
   text-transform: uppercase;
   letter-spacing: 3px;
   font-family: 'Montserrat', sans-serif;
-  color:white;
 }
 
-a:link{
+a:link, a:hover, a:focus, a:active{
   text-decoration: none;
```

Comment:
I think you can just include the text-decoration property above as 
```
.links a {
text-decoration: none;
}
```
then you don't need this or the below lines of
```
a:visited {
}
```
but I might be wrong